src/main/java/org/broadinstitute/hellbender/tools/exome/ExomeReadCounts.java

```diff
@@ -289,7 +295,8 @@ private PrintWriter openOutputWriter(final File output, final String headerText)
 
     @Override
     public void apply(final SAMRecord read, final ReferenceContext referenceContext, final FeatureContext featureContext) {
-        final GenomeLoc readLocation = genomeLocFactory.createGenomeLoc(read);
+
+        final SimpleInterval readLocation = new SimpleInterval(read.getContig(),read.getStart(),read.getEnd());
```

Comment:
I think this should be addressed in a different pull request as this one would not be the only place that  should be updated to use that new constructor.